### ISSUE TITLE
CorpusError when trying to run process_corpus.py

### ISSUE THREAD

**Issue by jabalazs** [epochx/opminreplicability#2]
After implementing the fixes mentioned in #1, I get the following error when trying to run `process_corpus.py`:

```
Traceback (most recent call last):
  File "process_corpus.py", line 51, in <module>
    corpus = Corpus()
  File "/home/weblab/opminreplicability/enlp/corpus/liu.py", line 39, in __init__
    raise CorpusError("Corpus was not properly built. Check for consistency")
enlp.corpus.utils.CorpusError: Corpus was not properly built. Check for consistency
```

Any ideas on how to fix it?

**Comment by epochx** [comment]
That's because the corpora come with some inconsistencies. Let's add a patch in the data download script to fix those.

**Comment by epochx** [comment]
Fixed in commit 46b332c0
